### ISSUE TITLE
GP: add transitions

### ISSUE THREAD

**Issue by MartinKolarik** [jsdelivr/www.jsdelivr.com#675]
The GP site has essentially no transitions for user actions/state changes.

**Comment by jimaek** [comment]
Do we need a designer here? Or Kiril could handle this after the more important tasks by himself?

**Comment by MartinKolarik** [comment]
@xbpcb what do you think?

**Comment by xbpcb** [comment]
Yeah, why not. I’ve been thinking about few possibilities for improving.

**Comment by MartinKolarik** [comment]
Ok, let's see where we can get.
